presto-main/src/main/java/com/facebook/presto/sql/gen/BytecodeGeneratorContext.java

```diff
@@ -33,15 +33,15 @@
     private final Scope scope;
     private final CallSiteBinder callSiteBinder;
     private final CachedInstanceBinder cachedInstanceBinder;
-    private final FunctionRegistry registry;
+    private final FunctionManager registry;
```

Comment:
Rename to `manager` or `functionManager` in all places. Search for `Manager registry` to find the instances